subsys/bluetooth/controller/Kconfig

```diff
@@ -447,9 +447,9 @@ config BT_CTLR_CHAN_SEL_2
 config BT_CTLR_ADV_EXT
 	bool "LE Advertising Extensions" if !BT_LL_SW_SPLIT
 	depends on BT_CTLR_ADV_EXT_SUPPORT
-	select BT_CTLR_SCAN_REQ_NOTIFY
-	# Uncomment when "LE Advertising Set Terminated event" is implemented
```

Comment:
I would prefer to wait until https://github.com/zephyrproject-rtos/zephyr/pull/26338 has been merged.